pkgs/development/python-modules/furl/default.nix

```diff
@@ -9,17 +16,27 @@ buildPythonPackage rec {
     sha256 = "08dnw3bs1mk0f1ccn466a5a7fi1ivwrp0jspav9arqpf3wd27q60";
   };
 
-  checkInputs = [ flake8 pytest ];
-
   propagatedBuildInputs = [ six orderedmultidict ];
 
-  # see https://github.com/gruns/furl/issues/121
-  checkPhase = ''
-    pytest -k 'not join'
-  '';
+  checkInputs = [
+    flake8
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+     # see https://github.com/gruns/furl/issues/121
+    "join"
+    # The following tests are failing on some installations
```

Comment:
Can you be more specific about the failures?

Reply:
Comment updated

Reply:
In a separate commit, that does not contain the changes it says it does, but just the comment. Please fix this up.